osu.Game/Screens/Play/HUD/GameplayLeaderboardScore.cs

```diff
@@ -244,6 +254,9 @@ protected override void LoadComplete()
 
         private void updateColour()
         {
+            if (HasQuit.Value)
+                return;
```

Comment:
Doing it this way has a quirk - if the user that quit was first on the leaderboard, their panel will still have a wider size, even as the user drops down in rankings naturally. I'd probably jam the "has quit" case in here (and rename the method, as it clearly does more than just update colour).